Documentation/docs.css

```diff
@@ -1,3 +1,67 @@
+.hamburger,
+.close-sidebar {
+  display: none;
+}
+
+.doc-container {
+  width: 100%;
+  margin: 3.125rem auto 0;
+  display: flex;
+  padding: 0 1rem;
+}
+
+#heading {
+  padding: 1rem 1.9rem;
+
+}
+
```

Comment:
a suggestion : 
use class instead of id (for header)
if feasible make variable for '1rem'
try make color into a variable , for #333

Reply:
Done